a.py

```diff
@@ -15,12 +15,16 @@ class newClass():
 
 	def __init__(self, x=None, y=None):
```

Comment:
```suggestion
	def __init__(self, x=None, y=None, *args, **kwargs):
```
So that function will not fail